pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -509,6 +511,28 @@ func (c *completedConfig) withUserAPIServer(delegateAPIServer genericapiserver.D
 	return server.GenericAPIServer, &legacyStorageVersionMutator{version: userapiv1.SchemeGroupVersion, storage: storage}, nil
 }
 
+func (c *completedConfig) withOpenAPIAggregator(delegatedAPIServer *genericapiserver.GenericAPIServer) (*genericapiserver.GenericAPIServer, error) {
```

Comment:
withOpenAPIAggregationController. Also remove the first return value. This is not functional, but mutates the input. Let's make that clear.